src/main/scala/vee/db/LevelDBWriter.scala

```diff
@@ -328,6 +353,17 @@ class LevelDBWriter(writableDB: DB, fs: FunctionalitySettings, val maxCacheSize:
     .recordStats()
     .build[(Int, BigInt), LeaseInfo]()
 
+
+  protected def loadLastWeightedBalance(address: Address): Option[Long] = readOnly { db =>
```

Comment:
it seems that this function is useless here, since, if we need to calculate minting average balance, i think call snapshot directly is much better than load it separately~